redaxo/src/addons/cronjob/lib/manager.php

```diff
@@ -98,7 +98,7 @@ public function log($success, $message)
             ($success ? 'SUCCESS' : 'ERROR'),
             ($this->id ?: '--'),
             $name,
-            strip_tags($message),
+            strip_tags(nl2br($message),'<br><b>'),
```

Comment:
ja - einfach escaped \n in das log... wird am saubersten sein. Aber die Umbrüche sind im Backend cronjob/log schon sehr hilfreich... manchmal wir es mit einer Zeile einfach zu unübersichtlich.